core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -50,14 +52,16 @@ class ReligionManager {
 
     fun isPickablePantheonBelief(belief: Belief): Boolean {
         if (belief.type != "Pantheon") return false
-        if (civInfo.gameInfo.civilizations.any { it.religionManager.pantheonBelief == belief.name })
+        if (civInfo.gameInfo.civilizations.any { it.religionManager.religion != null && it.religionManager.religion!!.pantheonBeliefs.contains(belief.name)})
             return false
         return true
     }
 
-    fun choosePantheonBelief(belief: Belief){
+    fun choosePantheonBelief(belief: Belief) {
         storedFaith -= faithForPantheon()
-        pantheonBelief = belief.name
+        religion = Religion(belief.name, civInfo)
+        religion!!.pantheonBeliefs.add(belief.name)
+        civInfo.gameInfo.religions[belief.name] = religion!!
```

Comment:
What will happen here, is that the first turn that this happens it will be passed by reference.
However, of you load from a saved game, the religion will be serialised into the save twice, and therefore will be deserislized twice

Reply:
Ah, that makes sense, hadn't thought of that